Nitrox.BuildTool/Models/GameInstallData.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using NitroxModel.Helper;
+
+namespace NitroxModel.Discovery;
+
+/// <summary>
+///     Game definition data gathered from Steam's files.
+/// </summary>
+public class GameInstallData
+{
+    private string managedDllsDir;
+    public string InstallDir { get; private set; }
+
+    public string ManagedDllsDir => managedDllsDir ??= Directory.EnumerateDirectories(InstallDir, "Managed", SearchOption.AllDirectories).FirstOrDefault();
+
+    private GameInstallData()
+    {
+        // Required for serialization
+    }
+
+    public GameInstallData(string installDir)
+    {
+        Validate.NotNull(installDir, $"Argument '{nameof(installDir)}' must not be null.");
+        InstallDir = installDir;
+    }
+
+    public static bool TryFrom(string path, out GameInstallData result)
+    {
+        try
+        {
+            GameInstallData game = new();
+            XmlDocument xDoc = new();
+            xDoc.Load(path);
+            XmlNamespaceManager nsManager = new(xDoc.NameTable);
+            nsManager.AddNamespace("d", xDoc.DocumentElement.NamespaceURI);
+            foreach (XmlElement elem in xDoc.DocumentElement.SelectNodes("//d:PropertyGroup/*", nsManager))
+            {
+                switch (elem.Name)
+                {
+                    case "GameDir":
+                        game.InstallDir = elem.LastChild.Value;
+                        break;
+                    case "GameManagedDir":
+                        if (!Directory.Exists(elem.LastChild.Value))
+                        {
+                            throw new DirectoryNotFoundException();
+                        }
+                        game.managedDllsDir = elem.LastChild.Value;
+                        break;
+                }
+            }
+
+            result = game;
+            return true;
+        }
+        catch (Exception)
```

Comment:
Is it intended we are not logging the exception? Maybe we want to filter them for expected errors.